src/graph.jl

```diff
@@ -16,17 +16,26 @@ end
 
 
 function polygon_array(coords::Vector{Vector{Vector{Vector{Float64}}}})::Array{LibGEOS.Polygon}
-    # Takes a complex vector object and returns an array of Polygons
-    # TODO(matthew): implement this, and add return type to signature
-    return []
+    """ Returns an array of Polygons for every entry in the `coords` array.
+    """
+    return LibGEOS.Polygon.(coords)
 end
 
 
 function min_bounding_rect(coords::Vector{Vector{Vector{Vector{Float64}}}})::Tuple
-    # Takes a complex vector object and returns an (axis-aligned) minimum
-    # bounding rectangle
-    # TODO(matthew): implement this, and add return type to signature
-    return ()
+    """ Takes a complex vector object and returns an (axis-aligned) minimum
+        bounding rectangle. A rectangle is represented by
+        ([lower_left_x, lower_left_y], [upper_right_x, upper_right_y]).
+    """
+    xmin, ymin , xmax, ymax = Inf, Inf, -Inf, -Inf
```

Comment:
So if I pass in an empty list of coords, I get a bounding box of infinite size? IDK if this is the intuitively correct behavior. I'd rather it return a bounding box of size 0 or just `nothing`. Alternatively, it could raise an error.

Reply:
Great catch. I think we should be raising an error than the other options because there is no such thing as a min bounding rect of [], and even a 0 would make no sense.

Reply:
Thanks for thinking about edge cases! Will throw an error here instead 😄